src/main/scala/codacy/detekt/DocGenerator.scala

```diff
@@ -84,15 +83,15 @@ object DocGenerator {
   }
 
   private def generateDescriptions(
-      rules: List[Rule],
-      descriptionsRoot: java.io.File,
-      extendedDescriptions: Map[String, String]): JsArray = {
+                                    rules: List[Rule],
+                                    descriptionsRoot: java.io.File,
+                                    extendedDescriptions: Map[String, String]): JsArray = {
```

Comment:
can we add our scalafmt formatting rules to this repo?

Reply:
Yes, I am going to add them.

Reply:
Scalafmt is checked in CI now.